### DISCUSSION TITLE
WARN: Unable to find Azure VM names for location: East US

### DISCUSSION THREAD

**Discussion by SashiTest7416** [nextflow-io/nextflow#2965] [Q&A] [answered]
We are not able to run nextflow on azure batch service.  Followed the article but we are not able to run. Please let us know if any setting to be changed.
![Capture](https://user-images.githubusercontent.com/107630240/174085985-6dc79e1e-c9d0-4436-a2a2-85dcf8922f84.PNG)

https://github.com/microsoft/Genomics-Quickstart/blob/main/03-Nextflow-Azure/README.md

Nextflow.config file

```nextflow
plugins {
  id 'nf-azure'
}

process {
  executor = 'azurebatch'
  container = 'nextflow/rnatoy@sha256:9ac0345b5851b2b20913cb4e6d469df77cf1232bafcadf8fd929535614a85c75'
}

azure {
  batch {
    location = 'East US'
    accountName = 'excelratest'
    accountKey = '$BATCH_ACCOUNT_KEY'
    autoPoolMode = true
    pools {
              centospool {
                  vmType = 'Standard_D4_v3'
                  maxVmCount = 2

              }
          }
}
storage {
    accountName = "excelrademostrg"
    accountKey = "$STORAGE_ACCOUNT_KEY"
  }
}

profiles {
  aztest {
    params.reads = "az://cbcrg-eu/ggal/*_{1,2}.fq"
    params.annot = "az://cbcrg-eu/ggal/ggal_1_48850000_49020000.bed.gff"
    params.genome = "az://cbcrg-eu/ggal/ggal_1_48850000_49020000.Ggal71.500bpflank.fa"
  }
}

```

**Comment by SashiTest7416** [comment]
HI Team, Any updates???

**Reply by GJSissons** [reply]
I just ran into the same issue

**Reply by abhi18av** [reply]
Could you please share the generated `.nextflow.log` file with us? 

Want to make sure it is not related to https://github.com/nextflow-io/nextflow/pull/2868

**Reply by GJSissons** [reply]
I was making the same error that @manuelesimi pointed out. I changed the location to "eastus" and it worked right away. The region shows as "East US" in the Azure portal, but that's only the display name. Thank you!!

**Comment by manuelesimi** [comment, marked as answer]
The location setting in your config file is not correct. You should use:
```
    location = 'eastus'
```
or 
```
    location = 'eastus2'
```
depending on your account's region in Azure.

See the name for each region here: 
https://azuretracks.com/2021/04/current-azure-region-names-reference/

**Comment by manuelesimi** [comment]
The instructions have been adjusted to reflect this discussion. See: https://github.com/nextflow-io/nextflow/pull/2991
They now explicitly mention the **name** of the region and how to get it.
